tools/diagnostics/service_bundle.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import os
+import retrying
+
+import sdk_cmd
+import sdk_diag
+from sdk_utils import groupby
+
+from diagnostics.bundle import Bundle
+import diagnostics.agent as agent
+
+log = logging.getLogger(__name__)
+
+DEFAULT_RETRY_WAIT = 1000
```

Comment:
Same as in the other file.

Reply:
3001569652caf2fb45657f1673bded4ec1556b4d